netlify.toml

```diff
@@ -42,3 +42,9 @@ ID = "Your_Site_ID"
   to = "/reference-capabilities/:splat"
   status = 301
   force = true
+
+[[redirects]]
+  from = "/pattern-matching/*"
+  to = "/expressions/:splat"
+  status = 301
+  force = true
```

Comment:
can you update this to be the specific urls to redirect? if we ever change these again, it will be to update again (potentially) and can you send the old overview/index to the match.md?

Reply:
Sure, not problem 👍🏻